specification/workloads/Workloads.SAPVirtualInstance.Management/main.tsp

```diff
@@ -15,6 +15,7 @@ import "./SAPDatabaseInstance.tsp";
 import "./SAPApplicationServerInstance.tsp";
 import "./SAPVirtualInstanceMetadata.tsp";
 import "./custom.tsp";
+import "../Workloads.Operations.Management/models.tsp";
```

Comment:
change is ok, just maybe weird that you import a file called models and this adds operations. It miught just call for cleaning up this shared folder

Reply:
> change is ok, just maybe weird that you import a file called models and this adds operations. It miught just call for cleaning up this shared folder

SDK Team trying to generate code from typespec directly, and as per the comment [here](https://github.com/Azure/sdk-release-request/issues/5539#issuecomment-2425446117), they are suggesting including common typespec project [Workloads.Operations.Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/workloads/Workloads.Operations.Management) in main typespec project [Workloads.SAPVirtualInstance.Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/workloads/Workloads.SAPVirtualInstance.Management), so that Operations List can also be included in the SDK Generation.

Can you help on this?

Reply:
> > change is ok, just maybe weird that you import a file called models and this adds operations. It miught just call for cleaning up this shared folder
> 
> After including the operation's models.tsp, all the checks related to SDK in this PR pipeline is failed. SDK Team trying to generate code from typespec directly, and as per the comment [here](https://github.com/Azure/sdk-release-request/issues/5539#issuecomment-2425446117), they are suggesting including common typespec project [Workloads.Operations.Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/workloads/Workloads.Operations.Management) in main typespec project [Workloads.SAPVirtualInstance.Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/workloads/Workloads.SAPVirtualInstance.Management), so that Operations List can also be included in the SDK Generation.
> 
> Can you help on this?

@timotheeguerin , gentle reminder on this

Reply:
i am not sure I get what is the ask here, the common seems to say you should have the common project as typespec, is this not the case?